patch/go.proto

```diff
@@ -16,6 +16,10 @@ message Options {
 	// For an enum value, this renames the generated Go const.
 	optional string name = 1;
 
+	// The embedded option changes the field as embedded in the generated Go struct.
+	// Only message types are allowed to be embedded, oneof fields are not embeddable.
+	optional bool embedded = 2;
```

Comment:
With fresh eyes, I think the verb form of `embed` makes sense here in the options API.

```suggestion
	optional bool embed = 2;
```